File: TsogosunProfileAdmin/MSProfileAdmin/Model/Dtos/PatronVoucherDto.cs
using System;

namespace tsogosun.com.MSProfileAdmin.Model.Dtos
{
    public class PatronVoucherDto
    {
        public long ID { get; set; }
        public long ParentVoucherID { get; set; }
        public string Vendor { get; set; }
        public string VoucherName { get; set; }
        public string VoucherFullDescription { get; set; }
        public int PointsAmount { get; set; }
        public string BurnPoints { get; set; }
        public int AmountAllowed { get; set; }
        public DateTime ExpiryDate { get; set; }
        public string DisplayOn { get; set; }
        public string OfferStatus { get; set; }
        public string TakeupMedium { get; set; }
        public DateTime? DatePrinted { get; set; }
        public string OfferAvailability { get; set; }
        public long? OtpNumber { get; set; }
       
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/profile-admin/patronVoucherDto.ts
export interface PatronVoucherDto {
    iD: number;
    parentVoucherID: number;
    vendor?: string | undefined;
    voucherName?: string | undefined;
    voucherFullDescription?: string | undefined;
    pointsAmount: number;
    burnPoints?: string | undefined;
    amountAllowed: number;
    expiryDate: Date;
    displayOn?: string | undefined;
    offerStatus?: string | undefined;
    takeupMedium?: string | undefined;
    datePrinted?: Date | undefined;
    offerAvailability?: string | undefined;
    otpNumber?: number | undefined;
}
